altair/utils/headless.py

```diff
@@ -130,8 +128,8 @@ def spec_to_image_mimebundle(spec, format, mode,
 
     Note
     ----
-    This requires the pillow, selenium, and chrome headless packages to be
-    installed.
+    This requires the pillow, selenium, and chromedriver or geckodriver
+    packages to be installed.
```

Comment:
Please make this more explicit: something like "this requires the pillow and selenium Python modules to be installed. Additionally it requires either chromedriver (if webdriver=='chrome')  or geckodriver (if webdriver=='firefox')"